docs/backends/quickwit.mdx

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
We can also add this destination to the [next-step.mdx](https://github.com/keyval-dev/odigos/blob/main/docs/next-steps.mdx)

Reply:
Done